.github/workflows/ci.yml

```diff
@@ -284,11 +288,25 @@ jobs:
         sudo cp $(go env GOPATH)/bin/zetcd /usr/local/bin/
 
         python3 -m pip install pytest-cov
+        tar -xf ./gae-${{ github.sha }}/vineyard.tar
         tar -xf ./gae-${{ github.sha }}/gae.tar
         tar -xf ./gle-${{ github.sha }}/gle.tar
-        sudo cp -R -n opt/graphscope/* /usr/local/
-        ls -la /usr/local/bin/
+
+        # install vineyard into /usr/local
+        sudo cp -r -n opt/vineyard/* /usr/local/
+
+        # link and install graphscope into /opt/graphscope
```

Comment:
To make it more precisely: "Install to /opt/graphscope and link them to /usr/local"